versioned/persist/tests/src/main/java/org/projectnessie/versioned/persist/tests/AbstractManyKeys.java

```diff
@@ -524,4 +524,56 @@ private static void checkKeysAndValuesIndividually(
           .isEqualTo(valueGen.apply(i).serialized());
     }
   }
+
+  public static Stream<List<String>> progressivelyManyKeyNames() {
+    // The seed of `7` was chosen after a few manual tries because it caused the most test failures
+    // in the key lookup code before the commit where this test was introduced.
```

Comment:
Can you add the PR number instead of "the commit" ?

Reply:
done